sotodlib/tod_ops/fft_ops.py

```diff
@@ -179,3 +179,29 @@ def find_superior_integer(target, primes=[2,3,5,7,11,13]):
         if (best_friend * base) <= best:
             best = best_friend * base
     return int(best)
+
+def calc_psd(aman, data=None, times=None, **kwargs):
+    """Calculates the power spectrum density of an input signal using signal.welch()
+    Data defaults to aman.signal and times defaults to aman.timestamps
+        Arguments:
+            aman: AxisManager with (dets, samps) OR (channels, samps)axes.
+
+            data: data signal to pass to scipy.signal.welch()
+            
+            times: timestamps associated with the data signal
+            
+            **kwargs: keyword args to be passed to signal.welch()
+
+        Returns two numpy arrays:
+            freqs:
+                array of frequencies corresponding to PSD calculated from welch
+            Pxx:
+                array of PSD values 
+    """
+    if data is None:
```

Comment:
Similarly, in accordance with #79 the "times" argument should be called "timestamps", since it takes its default from aman.timestamps.

Reply:
ah, so I switched "signal" to "data" because I import scipy.signal. Is there a naming convention I should follow? 

`signal.welch(...)` wouldn't know which "signal" (signal = aman.signal or scipy.signal) has the welch function, I believe.

Reply:
I think keep the signal keyword/override will be nice, the more functions we have that follow that convention is easier it will be to write code. You could do the import as: `from scipy.signal import welch`.